synapse/models/risk.py

```diff
@@ -101,86 +101,86 @@ def getModelDefs(self):
+                        'doc': 'The reporter\'s assessed location of the threat cluster.'}),
 
                     ('org:name', ('ou:name', {}), {
                         'ex': 'apt1',
-                        'doc': 'The name of the organization that the threat cluster is attributed to.'}),
+                        'doc': 'The reporter\'s name for the threat cluster.'}),
 
                     ('org:names', ('array', {'type': 'ou:name', 'sorted': True, 'uniq': True}), {
-                        'doc': 'An array of alternate names for the organization that the threat cluster is attributed to.'}),
+                        'doc': 'An array of alternate names for the threat cluster, according to the reporter.'}),
 
                     ('goals', ('array', {'type': 'ou:goal', 'sorted': True, 'uniq': True}), {
-                        'doc': 'The assessed goals of the threat cluster activity.'}),
+                        'doc': 'The reporter\'s assessed goals of the threat cluster.'}),
 
                     ('sophistication', ('meta:sophistication', {}), {
-                        'doc': 'The assessed sophistication of the threat cluster.'}),
+                        'doc': 'The reporter\'s assessed sophistication of the threat cluster.'}),
 
                     ('techniques', ('array', {'type': 'ou:technique', 'sorted': True, 'uniq': True}), {
                         'deprecated': True,
                         'doc': 'Deprecated for scalability. Please use -(uses)> ou:technique.'}),
 
                     ('merged:time', ('time', {}), {
-                        'doc': 'The time that this threat cluster was merged into another.'}),
+                        'doc': 'The time that the reporter merged this threat cluster into another.'}),
 
                     ('merged:isnow', ('risk:threat', {}), {
-                        'doc': 'The threat cluster that this cluster has been merged into.'}),
+                        'doc': 'The threat cluster that the reporter merged this cluster into.'}),
                 )),
                 ('risk:availability', {}, {}),
                 ('risk:tool:software:taxonomy', {}, ()),
                 ('risk:tool:software', {}, (
 
                     ('tag', ('syn:tag', {}), {
                         'ex': 'rep.mandiant.tabcteng',
-                        'doc': 'The tag used to annotate nodes that are part of the tool subgraph.'}),
+                        'doc': 'The tag used to annotate nodes that are associated with the tool.'}),
 
                     ('desc', ('str', {}), {
-                        'doc': "A description of the tool's use in threat activity."}),
+                        'doc': 'A description of the tool.'}),
 
                     ('type', ('risk:tool:software:taxonomy', {}), {
-                        'doc': 'An analyst specified taxonomy of software tool types.'}),
+                        'doc': 'A user-specified type for the tool.'}),
```

Comment:
Another "not necessarily user defined" spot where we should probably use a standard taxonomy type description